docs/getting-started-with-sylius/plugin-installation.rst

```diff
@@ -0,0 +1,40 @@
+Plugin installation
+===================
+
+Sylius is easy to customize to your business needs, but not all of the customizations you have to do on your own! Sylius
```

Comment:
Can we move the second Sylius to the new line?

Reply:
#secondSylius :D 🎉